repairnator/build-rainer/src/main/java/fr/inria/repairnator/TestServer.java

```diff
@@ -0,0 +1,86 @@
+package fr.inria.repairnator;
+
+import java.net.InetSocketAddress;
+import java.nio.ByteBuffer;
+import java.net.URI;
+import java.util.concurrent.TimeUnit;
+
+import org.java_websocket.WebSocket;
+import org.java_websocket.handshake.ClientHandshake;
+import org.java_websocket.server.WebSocketServer;
+
+//Test server for BuildRainer
+public class TestServer extends WebSocketServer {
```

Comment:
This `TestServer` is used as a stub for your test cases. I would suggest to move this file to `src/test` folder.